src/desktop/apps/article/components/InfiniteScrollNewsArticle.tsx

```diff
@@ -220,49 +179,43 @@ export class InfiniteScrollNewsArticle extends Component<
         if (hasMetaContent) {
           counter++
         }
-
-        const hasDateDivider = i !== 0 && this.hasNewDate(article, i)
         const isTruncated = !this.props.article || i !== 0
+        const hasDateDivider = i !== 0 && this.hasNewDate(article, i)
 
         return (
           <Fragment key={`article-${i}`}>
-            {hasDateDivider &&
-              <NewsDateDivider date={article.published_at} />
-            }
-            <div key={`article-${i}`}>
-              <Article
-                article={article}
-                isTruncated={isTruncated}
-                isMobile={isMobile}
-                marginTop={i === 0 ? marginTop : null}
-                onExpand={() => this.onExpand(article)}
-              />
-              <Waypoint
-                onEnter={(waypointData) => this.onEnter(article, waypointData)}
-                onLeave={(waypointData) => this.onLeave(i, waypointData)}
-                topOffset={FETCH_TOP_OFFSET}
-              />
-            </div>
-            {hasMetaContent && related && (
-              <Fragment>
-                <Break />
-                <RelatedArticlesCanvas
-                  articles={related}
-                  isMobile={isMobile}
-                />
-                <Break />
-              </Fragment>
-            )}
-            {hasMetaContent && displayAd && (
-              <Fragment>
-                <DisplayCanvas unit={displayAd.canvas} campaign={displayAd} />
-                <Break />
-              </Fragment>
-            )}
-          </Fragment >
+            {hasDateDivider && <NewsDateDivider date={article.published_at} />}
+            <NewsArticle
+              isMobile={isMobile}
+              article={article}
+              isTruncated={isTruncated}
+              isFirstArticle={i === 0}
+              onDateChange={(date) => this.onDateChange(date)}
+              nextArticle={articles[i + 1]}
+              onActiveArticleChange={(id) => this.onActiveArticleChange(id)}
+              isActive={activeArticle === article.id}
+            />
+            {hasMetaContent &&
+              related && (
```

Comment:
One thing I noticed while refactoring the Reaction article layouts, is that we add the metadata on the Reaction side rather than in Force.  Not for this PR, but we might consider moving the below fragment into Reaction's NewsLayout.